content/2017/03/30/safe-get.rst

```diff
@@ -0,0 +1,257 @@
+public: yes
+author: miriam
+headline:
+  - tagline: 'exploring safe-get-function and safe-call utilities'
+    type: 'Code Sample'
+tags: [
+  Sass,
+  Tools,
+  Code
+  ]
+image:
+  - src: '2017/ga-spam/spam-traffic.jpg'
+summary: |
+  The `Sass 3.5 Release Candidate`_
+  includes support for
+  first-class functions.
```

Comment:
Right, it's defined below, but it's not actually a link here. It's missing the ` and _.